affiliatewp-order-details-for-affiliates.php

```diff
@@ -170,35 +197,40 @@ private function hooks() {
 	}
 
 	/**
-	 * Redirect affiliate to main dashboard page if they cannot access order details tab
+	 * Redirect affiliate to main dashboard page if they cannot access order details tab.
 	 *
 	 * @since 1.0
 	 *
 	 * @return void
 	 */
 	public function no_access() {
 		if ( $this->is_order_details_tab() && ! ( $this->can_access_order_details() || $this->global_order_details_access() ) ) {
-			wp_redirect( affiliate_wp()->login->get_login_url() ); exit;
+			wp_redirect( affiliate_wp()->login->get_login_url() );
+			exit;
 		}
 	}
 
 	/**
-	 * Whether or not we're on the customer's tab of the dashboard
+	 * Whether or not we're on the customer's tab of the dashboard.
```

Comment:
```suggestion
	 * Determines whether or not we're on the customer's tab of the dashboard.
```
Third-person singular verb in a DocBlock summary.